### DISCUSSION TITLE
Any way to collect only specific metrics

### DISCUSSION THREAD

**Discussion by andrii-bodnar** [martin-helmich/prometheus-nginxlog-exporter#275] [Q&A]
Hey there,

Is there any way to collect only specific metrics?

For example, I do not need the `*_hist_bucket`, `*_hist_sum`, `*_hist_count` metrics. Is there some configuration to disable collecting these metrics?

I know that it could be filtered on the Prometheus side, but I just want the exporter to do the only needed work.

Thanks in advance!

**Comment by martin-helmich** [comment]
There is no way to disable the exporter from exporting specific metrics.

You can discard specific metrics (and more) when scraping them in Prometheus. Have a look at Prometheus' [`relabel_config` documentation](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#relabel_config) for more information.
